briefcase/app.py

```diff
@@ -262,6 +266,75 @@ def install_code(self):
                 '.'
             ])
 
+    @property
+    def launcher_header(self):
+        """
+        Override the shebang line for launcher scripts
+        This should return a suitable relative path which will find the
+        bundled python for the relevant platform
+        """
+        return None
+
+    @property
+    def launcher_script_location(self):
+        return self.app_dir
+
+    def install_launch_scripts(self):
+        exe_names = []
+        if self.distribution.entry_points:
```

Comment:
What's the behavior here if the user has an app with multiple entry points, which *will* make sense for desktop deploys, but *won't* make sense for mobile? 

Would it make sense to roll out a full mobile template for each app? (and, for that matter - for macOS too, where the alternate entry points will be buried)

Reply:
I've gone back and forth on this one myself. It's both a pro and con of the packaged app formats that you can't share any bulk between them.

Initially I baulked at the idea of producing isolated copies of the entire app just for the sake of a modified startup script, however now I think it's probably the way to go. 
It ultimately gives the end user the option of which of the produced apps they want to install, and not worry about any library dependencies, for the sake of a few more MB of disk space. 

This would be a significant extra piece of work that I'd prefer to do in a follow up PR if that's ok. It'll mean a bit of restructuring of the main workflow in app, checking for entry points early on and if so probably looping over the rest of the workflow with some optiomisations to minimise duplication of common calculations like building the main apps bdist.

This will also be needed on linux if it moves to snappy/flatpack.

Reply:
As far as current behaviour is concerned.... on mobile releases a defined entry point with the same name as the package formal_name will override the hardcoded entrypoint in the template, otherwise it will essentially be ignored and the template method will be used.

Reply:
Sounds good - tackling this as a follow up PR works for me.